### PR TITLE
loki datasource

### PR DESCRIPTION
**Pull request by buixor**

### CODE REVIEW

File: crowdsec-docs/docs/data_sources/loki.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+This will allow to read logs from loki, using the query `{job="varlogs"}`.
+```yaml
+source: loki
+log_level: info
+url: http://localhost:3100/
+limit: 1000
+query: |
+  {job="varlogs"}
+auth:
```
Comment (LaurenceJJones) [inline crowdsec-docs/docs/data_sources/loki.md:18]: Does acquisition support variables?

Comment (buixor) [reply crowdsec-docs/docs/data_sources/loki.md:18]: wdym ?

Comment (LaurenceJJones) [reply crowdsec-docs/docs/data_sources/loki.md:18]: You might not to store authentication credentials as plain text

**Comment by LaurenceJJones** [comment]
Please merge main as docusarus v3 has been integrated

**Review by blotus — APPROVED** [review]

**Pull request merged**